README.md

```diff
@@ -43,192 +47,58 @@ MILV's logic is based on this basic distinction:
 | `-base-path`                   | Root directory of the repository                            | `""`               |
 | `-backoff`                     | Backoff timeout                                             | `"1s"`             |
-| `-config-file`                 | Configuration file for the bot. See the [**Config file**](#configuration-file) section for more details.  | `milv.config.yaml` |
-| `-external-links-to-ignore`    | Comma-separated external links which will not be checked    | `[]`               |
-| `-internal-links-to-ignore`    | Comma-separated internal links which will not be checked    | `[]`               |
-| `-files-to-ignore`             | Comma-separated files which will not be checked             | `[]`               |
-| `-allow-redirect`              | Redirects will be allowed                                   | `false`            |
+| `-config-file`                 | Configuration file for the bot. See the [**Configuration file**](/docs/configuration-file.md) for more details.  | `milv.config.yaml` |
+| `-external-links-to-ignore`    | Comma-separated external links which MILV must not check    | `[]`               |
+| `-internal-links-to-ignore`    | Comma-separated internal links which MILV must not check    | `[]`               |
+| `-files-to-ignore`             | Comma-separated files which MILV must not check            | `[]`               |
+| `-allow-redirect`              | Redirects should be allowed                                   | `false`            |
 | `-request-repeats`             | Number of repeated request                                  | `1`                |
-| `-allow-code-blocks`           | Links in code blocks will be checked                        | `false`            |
+| `-allow-code-blocks`           | Validating links in code blocks should be allowed                        | `false`            |
 | `-timeout`                     | Connection timeout (in seconds)                             | `30`               |
-| `-ignore-external`             | External links to be ignored                                | `false`            |
-| `-ignore-internal`             | Internal links to be ignored                                | `false`            |
+| `-ignore-external`             | External links that MILV must ignore                                | `false`            |
+| `-ignore-internal`             | Internal links that MILV must ignore                                 | `false`            |
 | `-v`                           | Verbose logging                                             | `false`            |
-| `-help` or `-h`                | Available parameters                                        | n/a                |
+| `-help` or `-h`                | Available parameters                                        |  n/a                |
 
 Files to be checked are given as free parameters.
 
-### Configuration file
```

Comment:
Maybe we could leave this section mentioning the configuration file in the main README but only explain that it's possible to configure MILV with config and link the document with details?

I think using the config file is the most common way to use MILV and if we only mention it in the description of command line parameter it's possible that someone who sees the project for the first time might miss it.

Reply:
You are absolutely right :)